tests/integration/test_integratord/test_integration_options/data/configuration_templates/config_pagerduty_no_option_tag.yaml

```diff
@@ -0,0 +1,35 @@
+- tags:
+    - all
+  apply_to_modules:
```

Comment:
Maybe these tags are not necessary now, because there are on config per file

Reply:
Done!